.gitignore

```diff
@@ -1,3 +1,4 @@
 pacbot_server
 **/node_modules
-**/__pycache__/
\ No newline at end of file
+**/__pycache__/
+**/.idea
```

Comment:
I didn't expect it to be necessary to add IDE-specific files to the .gitignore, but let's discuss this at the meeting.